### DISCUSSION TITLE
Workflows do not have access to defined variables when runnning by dependabot

### DISCUSSION THREAD

**Discussion by neilime** [community/community#44088] [Actions]
### Select Topic Area

Bug

### Body

I just started to use [Configuration Variables](https://github.blog/changelog/2023-01-10-github-actions-support-for-configuration-variables-in-workflows/), and it's pretty cool and useful +1

In my case, I'm using it for centralizing duplicated configurations across the organisation (e.g. a OCI registry url...) and repositories non sensible settings widely used in workflows (deployment urls...)

![Screenshot](https://user-images.githubusercontent.com/314088/212033833-594bb3d7-cf84-4ba9-8398-26d5dc9fd531.png)

Is is working like a charm and avoid hard coding these variables in yml files.

BUT, it looks like dependabot did not have access to these variables, so every workflow triggered by dependabot (e.g. pull request checks)

Example with variable "OCI_REGISTRY", missing.

- Workflow:

![config](https://user-images.githubusercontent.com/314088/212049587-6654dbe8-a6c1-4bbd-abcb-576b68be5284.png)

- Logs: 

![logs](https://user-images.githubusercontent.com/314088/212039875-1191f3d4-8f59-45a5-a310-d2234a0d2722.png)

Is there anything to configure to make it works, I am doing something wrong?

**Comment by tbouffard** [comment]
See also #44322: this is the same behavior for Pull Request created from a fork repository.

**Comment by dym-ok** [comment]
Yes, it is surprising how variables have only made it to the `Actions` subsection of `Secrets and variables`, but not to `Codespaces` or `Dependabot`.

I wonder what's an ETA for this could be? Otherwise it's really difficult to use variables in any PR-related workflows if Dependabot scanning is enabled.

**Comment by MarkIannucci** [comment]
@ashokirla, I thought I'd surface this discussion since you wrote the [blog post](https://github.blog/2023-01-10-introducing-required-workflows-and-configuration-variables-to-github-actions/) announcing this functionality.  Does GitHub have plans to make the configuration variables accessible to dependabot and codespaces?

**Reply by ashokirla** [reply]
Hi @MarkIannucci , yes we already have this ask in our backlog and will look into it this quarter.

**Comment by tauhid621** [comment]
👋 The changes to support Actions Variables for dependabot actor PRs was deployed. Every workflow triggered by dependabot actor would have access to Actions Variables.

The docs would be updated soon.

**Reply by isaacsanders** [reply]
Is this on a released version of GHES? Are there release notes to refer to?

**Reply by mikelax** [reply]
> 👋 The changes to support Actions Variables for dependabot actor PRs was deployed. Every workflow triggered by dependabot actor would have access to Actions Variables.
> 
> The docs would be updated soon.

I know this is an old thread but I am not seeing the docs about variables reflected of this change. It would be helpful as I just spent some time trying to figure out of dependabot has access to Org level variables.
